enclave/core/sgx/exit.S

```diff
@@ -100,6 +119,9 @@ oe_asm_exit:
     // simulate-flag-check.
     lfence
 
+    // Clear %r9 which was being used to maintain td pointer
+    xor %r11, %r11
```

Comment:
This needs to be `%r9' I think. Can you please thoroughly recheck the register usage?

Reply:
Fixed this one and another mistake I found in enter.S. I'll go through again today to make sure everything looks right.